docs/examples/droplet_fusion/droplet_fusion.rst

```diff
@@ -0,0 +1,200 @@
+
+Select data for fit
+^^^^^^^^^^^^^^^^^^^
+Below we are selecting the force and trap data at the fusion event. When fitting the fusion relaxation time, 
+it is important that the traps holding the droplets are either both static, or one of the traps is moving at a constant speed. 
+Therefore, we plot the trap position over time::
+
+    start = "5.4s"
+    stop = "5.9s"
+
+    force_selection = f.force2x[start:stop]
+    trap_selection = f["Trap position"]["1X"][start:stop]
+
+    plt.figure()
+    plt.subplot(2, 1, 1)
+    force_selection.plot()
+    plt.ylabel("Laser signal (a.u.)")
+    plt.subplot(2, 1, 2)
+    trap_selection.plot()
+    plt.ylabel("x-coordinate ($\mu$m)")
+    plt.tight_layout()
+    plt.show()
+
+.. image:: selected_data.png
+
+Model for fusion
+^^^^^^^^^^^^^^^^
+
+We will be fitting the force data during the fusion event with the following equation: $f(t) = ae^{-t/\tau}+bt+c$                 
+
+The term $bt$ accounts for the movement of the trap, assuming a constant trap speed [1]_. The parameter of interest is $\tau$, the relaxation time schale of the fusion event::
+
+    from scipy.optimize import curve_fit
+
+    def relaxation_model(t, tau, a,b,c):
+        return a*np.exp(-t / tau)+b*t+c
+
+Fit the data and plot the result::
+
+    time = force_selection.seconds
+    force = force_selection.data
+
+    popt, pcov = curve_fit(relaxation_model, time, force, [0.1,force[0],0,0])
+    print(popt)
+    plt.figure()
+    plt.plot(time,force)
+    plt.plot(time,relaxation_model(time,*popt),label=f'$\\tau$ = {popt[0]:0.2f}s')
+    plt.legend()
+    plt.show()
+
+.. image:: fit.png
+
+The relaxation time obtained from the fit is 0.05 seconds. 
+
+Now we will proceed to determined the size of the droplets before the fusion event.
+
+Droplet size
+------------
+
+First load the scan and print the relevant metadata::
+
+    >>> for name, scan in f.scans.items():
+            print(f"num frames: {scan.num_frames}")
+            frame_duration = (scan.frame_timestamp_ranges()[0][1]-scan.frame_timestamp_ranges()[0][0])/1e9
+            print(f"frame duration: {frame_duration} s")
+
+    num frames: 27
+    frame duration: 0.4977664 s
+
+Plot a frame before the fusion event::
+
+    framenr = 2
+    plt.figure()
+    scan.plot(channel='green',frame=framenr)
+    plt.show()
+
+.. image:: frame2.png
+
+Plot a frame after the fusion event::
+
+    framenr = 6
+    plt.figure()
+    scan.plot(channel='green',frame=framenr)
+    plt.show()
+
+.. image:: frame6.png
+
+If the droplets are in focus, the size of the droplet can be estimated from the 2D scan. 
+First, we use image segmentation to identify the two droplets in the image::
+
+    import math
+    from skimage.measure import label, regionprops
+
+    framenr = 2  # Choose a frame before the fusion event on which you want to identify and measure droplets
+
+    image = scan.get_image(channel='red')[framenr]
+    image = image / np.max(image)
+    threshold = 0.5
```

Comment:
Interesting. I wonder to what extent the determined size depends on this threshold.

One thing I wonder about is the normalization on a per image basis. How does this maximum change over time?

How would it affect size estimates if you were to normalize the whole stack at once and just use one threshold for all the images?

Reply:
That is a good point. First of all, I don't expect the size determination to be perfect. Scan images are always a bit 'blurry' and I expect the error in the size to have a ~10% error. There might be better choices for the threshold, that are closer to the real size. I found 50% a good 'null' estimate. In the end, the size has to be determined for many separate scans from different fusion events. The final method has to be insensitive to intensity fluctuations between different scans. I expect the variation between scans to be much larger than within a scan. I could try different values for the threshold and normalization, and see whether the final estimated is affected a lot.

Reply:
I tried different values for the threshold and normalization, and the current choice actually works quite well. (some extreme values eg result in detecting more than two, or just 1 object). I added a remark that the error is ~10%, which accounts for the range of values I got for the radii.

Reply:
And I realized that the drops are not spherical, because they move while the scan is being recorded... This adds another layer of uncertainty for the droplet size...

Reply:
Your last point is a good one.

Well, considering that in the dataset, the fast axis is horizontal, wouldn't it make sense to specifically look at the largest horizontal diameter then (least amount of error) rather than find an ellipse?

Then you also don't run the risk of comparing different orientations with eachother.

If you ensure that the width you compute is fast axis-aligned, you could even correct for the motion blur exactly! My guess is the precision of such an estimate would be slightly lower (since you're not averaging as much), but the accuracy would be higher (lower bias).

Maybe something like this would be an option?
```python
fig, ax = plt.subplots()
ax.imshow(image, cmap=plt.cm.gray)
average_droplet_radii = np.array([])  # List of arrays to collect all droplet radii

# For each label, find the maximal extent in the direction of the fast axis
# This results in the least error due to movement artefacts
def get_center_and_width(scan, mask, axis):
    """Grabs the center and width along the fast scanning axis"""
    widths = np.sum(mask, axis=axis)
    max_width = np.max(widths)

    # Grab the position
    coordinate_weighted_mask = np.indices(mask.shape)[axis] * mask
    centers = np.sum(coordinate_weighted_mask, axis=axis) / np.clip(np.sum(mask, axis=axis), 1, np.inf)
    
    # Since some scanlines can have the same width, we'd want the vertical position to be the average of these
    max_scanline = int(np.mean(np.nonzero(max_width == widths)[0]))

    if axis:
        center = (centers[max_scanline], max_scanline)
    else:
        center = (max_scanline, centers[max_scanline])
    
    return center, max_width


def plot_width(scan, center, width, axis):
    plt.plot(center[0], center[1], "ko")
    if axis:
        plt.plot([center[0] - 0.5 * width, center[0] + 0.5 * width], [center[1], center[1]])
    else:
        plt.plot([center[0], center[0]], [center[1] - 0.5 * width, center[1] + 0.5 * width])


ax.imshow(image, cmap=plt.cm.gray)
axis = 1 if scan.fast_axis == "X" else 0
center, width = get_center_and_width(scan, label_img == 1, axis)
plot_width(scan, center, width, axis)
center, width = get_center_and_width(scan, label_img == 2, axis)
plot_width(scan, center, width, axis)

plt.title("Fast scanning axis aligned width")
plt.show()
```

<img width="547" alt="image" src="https://github.com/user-attachments/assets/2f642b3f-b623-4f5c-bf81-fd7200e82a61">

Reply:
I like that suggestion, will add it!

Reply:
Awesome. I did do a quick estimate of the kind of error one might expect from the movement, but when aligned on the fast axis like this, it's at the subpixel level and I think it's really completely negligible compared to the potential error we'd be making already with thresholding the image anyhow. I get numbers on the order of 0.01 pixels.

Reply:
Great, thanks. I adjusted the comment and now I write that the uncertainty of the sizes is caused by the blurryness of the image (and not anymore by the movement of one of the droplets)